ironfish-cli/src/commands/accounts/transactions.test.ts

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { expect as expectCli, test } from '@oclif/test'
+import { GetTransactionsResponse } from 'ironfish'
```

Comment:
This import didn't work for me, I had to change the path to  `../../../../ironfish/src/rpc/routes/transactions`

Reply:
yup, this needs updating for the latest version. I just pushed an update.